compose-builder/add-security.yml

```diff
@@ -201,17 +172,28 @@ services:
     tmpfs:
       - /run
       - /tmp
-    command: >
-      /bin/sh -c 
-      "until /consul/scripts/consul-svc-healthy.sh kong-migrations; do sleep 1; done;
-      /docker-entrypoint.sh kong docker-start"
+    command: '
```

Comment:
nitpick: suggest following the YAML multi-liner `>` like the original one does. See https://yaml-multiline.info/

Reply:
I originally did it that way, but it had some conflicts...

Reply:
Note that docker-compose is reformatting this for the generated files. I preferer the most readable formatting here.

Reply:
Soooo does that mean I need to change it or leave it as is? :-)

Reply:
I am fine with it as is. @jim-wang-intel ?

Reply:
> I originally did it that way, but it had some conflicts...

What're conflicts?   once you give the `>`  after command: the next line should be just do whatever contents of command you want to do.

Reply:
> What're conflicts? once you give the `>` after command: the next line should be just do whatever contents of command you want to do.

I don't remember specifically, but when I ran the compose file when formatted the other multi-line way, it wouldn't let me run the compose file at all - like it conflicted with the general format of the file. I can try it again...gimme a few...

Reply:
@jim-wang-intel @lenny-intel I just updated, tested, and pushed. Maybe I had something screwy before...looks like it works now...